src/ngraph_builder.cc

```diff
@@ -3937,6 +3937,45 @@ static Status TranslateTileOp(
   return Status::OK();
 }
 
+// Translate TopKV2 Op using ngraph core op TopK
+static Status TranslateTopKV2Op(
+    const Node* op, const std::vector<const Tensor*>& static_input_map,
+    Builder::OpMap& ng_op_map) {
+  shared_ptr<ngraph::Node> ng_input;
+  ValidateInputCount(op, 2);
+  TF_RETURN_IF_ERROR(GetInputNode(ng_op_map, op, 0, &ng_input));
+
+  size_t k_axis = ng_input->get_shape().size() - 1;
+
+  std::vector<int32> ng_k;
+  size_t k;
+  bool sorted = true;
+
+  TF_RETURN_IF_ERROR(GetStaticInputVector(op, 1, static_input_map, &ng_k));
+  k = ng_k[0];
+
+  GetNodeAttr(op->attrs(), "sorted", &sorted);
+
+  // ngraph topk computes topk min if sorted is false
+  // TODO:ngraph team need to add an attribute to always compute max for topkv2
+  if (sorted == false) {
+    return errors::InvalidArgument("topkv2 should compute max, not min");
+  }
+
+  shared_ptr<ngraph::Node> ng_result = make_shared<ngraph::op::TopK>(
+      ng_input, k_axis, ng::element::i32, k, sorted);
```

Comment:
Should we fix the `ng::element::i32` here, this is for input data type or sth else's data type?

Reply:
this is for the index element type which is always an int, so it could be int32 or int34.

Reply:
Output of TF's topkv2 op is int32 (https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/nn_ops.cc#L1250), `so ng::element::i32` is fine here